### PR TITLE
Fixes issue #39 (remove draw function from from how-to's and tutorials)

### PR DESCRIPTION
**Pull request by HaimHorowitzAgnostiq**
<!--
I was able to remove the draw functions from the how-to's but not from the tutorials due to the following problems: (1) The dispatch ID won't won't show in the browser, and (2) the code is broken in the nitrogen tutorial 
-- >

- [ ] I have added the tests to cover my changes.
- [ ] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [ ] I have read the CONTRIBUTING document.

### CODE REVIEW

**Review by FyzHsn — CHANGES_REQUESTED** [review]
@HaimHorowitzAgnostiq None of the images are rendering on the documentation that was built. Can you investigate why?
Screen Shot 2022-03-01 at 11 44 45 AM

Also, can you add a single line explaining that the user needs to go to the browser UI to check the workflow lattice in the visualize lattice nb.

The images aren't rendering in any of the how-tos

**Comment by FyzHsn** [comment]
@HaimHorowitzAgnostiq There's a quite a few issue here. I'd recommend leaving this as is and later someone else can pick up on this.

**Comment by FyzHsn** [comment]
@kessler-frost Can you change the distance linspace spacing to 4 instead of 7 in the quantum chemistry tutorial. That should temporarily solve the problem encountered in this [issue](https://github.com/AgnostiqHQ/covalent/issues/193).

File: doc/source/tutorials/quantum_gravity/spacetime_classification.ipynb [hunk 1]
```diff
@@ -7,6 +7,8 @@
    "source": [
     "## Classifying discrete spacetimes by dimension\n",
     "\n",
+    "Note: Breaks on M1 Macs because of `tensorflow` causing kernel panic.\n",
```
Comment (scottwn) [inline doc/source/tutorials/quantum_gravity/spacetime_classification.ipynb:10]: can you also add this to Known Issues in the README?

**Review by scottwn — CHANGES_REQUESTED** [review]
LGTM, just please add Known Issues back to the README and add the note about Tensorflow on M1

**Review by FyzHsn — APPROVED** [review]

File: doc/source/tutorials/quantum_gravity/spacetime_classification.ipynb (hunk 1 above)
Comment (kessler-frost) [reply doc/source/tutorials/quantum_gravity/spacetime_classification.ipynb:10]: Added to Known Issues in README.

**Review by scottwn — APPROVED** [review]

**Pull request merged**